schemas/MST_mirror_2f_measurements.schema.yml

```diff
@@ -1,44 +1,39 @@
+%YAML 1.2
 ---
-# Follows schema https://github.com/gammasim/workflows/schema/jsonschema.yml
-title: Mirror panel 2F measurements description
-description: |-
-  Data description of mirror panel 2F measurements used as
-  input to SimPipe.
-name: simpipe-schema
+title: Schema for mirror panel 2F measurements
 version: 0.1.0
-
-schema:
-
-  - name: mirror_2f_measurement
-    description: |-
-      Mirror panel 2F measurements determining
-      panel radius and panel PSF.
-    data:
-      - type: datatable
-        table_columns:
-          - name: mirror_panel_id
-            description: |-
-              Mirror panel ID.
-            required_column: true
-            units: dimensionless
-            type: string
-          - name: mirror_panel_radius
-            description: |-
-              Mirror panel radius.
-            required_column: true
-            units: cm
-            type: double
-          - name: psf
-            description: |-
-              Spot size of mirror panel PSF at nominal distance.
-            required_column: true
-            units: cm
-            type: double
-          - name: psf_opt
-            description: |-
-              Spot size of mirror panel PSF
-              (best value, not at nominal distance).
-            required_column: false
-            units: cm
-            type: double
-...
+base_schema: simpipe-schema
+base_schema_url: https://github.com/gammasim/workflows/schema/jsonschema.yml
+base_schema_version: 0.1.0
+name: mirror_2f_measurement
+description: |-
+  Mirror panel 2F measurements determining
+  panel radius and panel PSF.
+data:
+  - type: datatable
+    table_columns:
+      - name: mirror_panel_id
+        description: |-
+          Mirror panel ID.
+        required: true
+        units: dimensionless
+        type: string
+      - name: mirror_panel_radius
+        description: |-
+          Mirror panel radius.
+        required: true
+        units: cm
+        type: double
+      - name: psf
+        description: |-
+          Spot size of mirror panel PSF at nominal distance.
+        required: true
+        units: cm
+        type: double
+      - name: psf_opt
+        description: |-
+          Spot size of mirror panel PSF
```

Comment:
please refer in the description also what "opt" means.

Reply:
This is a first example, let's move the discussion on column names somewhere else. This is what we have used in the whole simtelarray/model validation until now and is a first example.